main.go

```diff
@@ -57,6 +57,7 @@ func init() {
 	flag.StringVar(&cacert, "ca-cert", env("ETCD_CACERT", ""), "CA Certificate")
 	flag.DurationVar(&config.ReadTimeout, "rtimeout", 2*time.Second, "read timeout")
 	flag.BoolVar(&config.RoundRobin, "round-robin", true, "round robin A/AAAA replies")
+	flag.BoolVar(&config.NSRotate, "ns-rotate", true, "round robin selection of nameservers from among those listed, rather than have all forwarded requests try the first listed server first every time.")
```

Comment:
Just stop the explanation of the word "listed", otherwise it becomes rather longish...